services/rekognition.cfc

```diff
@@ -87,6 +87,31 @@ component {
         return apiCall( requestSettings, 'RecognizeCelebrities', { 'Image': arguments.Image } );
     }
 
+    /**
+    * Returns an array of face matches ordered by similarity score in descending order
+    * https://docs.aws.amazon.com/rekognition/latest/dg/API_CompareFaces.html
+    * @SourceImage a struct with a "Bytes" key containing Base64-encoded binary data or an "S3Object" struct containing a "Bucket", "Key", and optional "Version" - https://docs.aws.amazon.com/rekognition/latest/dg/API_Image.html
+    * @TargetImage a struct with a "Bytes" key containing Base64-encoded binary data or an "S3Object" struct containing a "Bucket", "Key", and optional "Version" - https://docs.aws.amazon.com/rekognition/latest/dg/API_Image.html
+    * @SimilarityThreshold a numeric minimum level of confidence that a match must meet to be included. Valid Range: Minimum value of 0. Maximum value of 100.
+    * @QualityFilter a string filter that specifies a quality bar for how much filtering is done to identify faces. Valid Values: NONE | AUTO | LOW | MEDIUM | HIGH
+    */
+    public any function compareFaces(
+        required struct SourceImage,
+        required struct TargetImage,
+        numeric SimilarityThreshold,
+        string QualityFilter
+    ) {
+        var requestSettings = api.resolveRequestSettings( argumentCollection = arguments );
+        var args = { 
+            'SourceImage': arguments.SourceImage,
+            'TargetImage': arguments.TargetImage
+        };
+        if ( !isNull( arguments.SimilarityThreshold ) ) args[ 'SimilarityThreshold' ] = arguments.SimilarityThreshold;
+        if ( !isNull( arguments.QualityFilter ) ) args[ 'SimilarityThreshold' ] = arguments.QualityFilter;
```

Comment:
This should probably be `args[ 'QualityFilter' ]`.

Reply:
You're right! Now it's fixed. https://github.com/jcberquist/aws-cfml/pull/38/commits/b45eb5cb4118bc7c16e7bb3f4e5aaf751a4e0f97